apps/app-router/e2e/app-router-funnel.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { test, expect } from '@playwright/test';
+
+const APP_ROUTER_LOCAL_URL = 'http://localhost:3100/';
+
+test('can move the steps of the funnel using history.push.', async ({ page }) => {
+  await page.goto(APP_ROUTER_LOCAL_URL);
+
+  await expect(page.getByText('start')).toBeVisible();
+
+  await expect(page.getByRole('button', { name: 'next' })).toBeVisible();
+
+  await page.getByRole('button', { name: 'next' }).click();
+
+  await expect(page.getByText('end')).toBeVisible();
```

Comment:
It would be nice to have a test case for browser "history back".

Reply:
🔥 That's a good idea! I will also add a test case for history.back.